src/rofl/datapath/pipeline/openflow/openflow1x/pipeline/of1x_flow_table.h

```diff
@@ -172,6 +172,13 @@ rofl_result_t __of1x_destroy_table(of1x_flow_table_t* table);
 */
 
 /**
+* @internal Add flow entry to table being able to match cookie
+*
+* @warning Out of spec.
```

Comment:
Basically its enough to add something like you have written in your comment here :)